docs/de/contribute/README.md

```diff
@@ -0,0 +1,128 @@
+
+Der Verein [busFaktor() e.V.](https://busfaktor.org/de/) koordiniert und fördert die Programmierung von *ocelot.social*.
+Unterstütze die Nachhaltigkeit  der Software und deren Netzwerke gerne mit einer Einmal- oder Dauerspende:
+
+<a href="https://busfaktor.org/de/spenden" target="_blank">
+  <Button class="donate-button">
+    Hier spenden
+  </Button>
+</a>
+
+### Unterstütze dein Netzwerk
+
+#### Nutze die aktiven Netzwerke
+
+Sei mit dabei und werde Nutzer eines Netzwerks, das mit unserer Software läuft.
+
+- Registriere dich als Nutzer.
+- Ergänze dein Benutzerprofil mit einem Profilbild, deinem ungefähren Standort und weiteren Angaben über dich.
+- Schreibe lesenswerte und lesefreundliche Beiträge. Ergänze sie mit einem Bild und mit Hashtags.
+- Kommentiere die Beiträge anderer Nutzer.
+- Empfehle lesenswerte Beiträge.
+- Folge Nutzern.
+- Gründe eine Gruppe und lade Mitglieder ein.
+- Trage dazu bei, dass dein Netzwerk ein angenehmer und interessanter Ort für dich und deine Freunde ist.
+
+#### Belebe dein Netzwerk
+
+Bist du Mitglied eines Netzwerks, dann möchtest du dort interessanten Menschen begegnen.
+Mit mehr Nutzern wird dein Netzwerk dynamisch und vielfältig.
+
+- Erzähle deinen Freunden und Verwandten vom Netzwerk.
+- Lade sie ins Netzwerk ein (→ [Anleitung im Wiki](https://github.com/Ocelot-Social-Community/Ocelot-Social/wiki/de:Invitations)).
+- Veröffentliche den Link zum Netzwerk auf deiner Website, deinem Blog oder in anderen sozialen Netzwerken.
+- Berichte vor Publikum vom Netzwerk.
+
+#### Weitere Möglichkeiten dich zu engagieren
+
+Viele Netzwerke sind Mitmach-Netzwerke und sind dankbar für deine Hilfe – vielleicht als Moderator?
+Für weitere Informationen besuche die Website des Betreibers oder kontaktiere ihn.
+
+### Betreibe ein eigenes Netzwerk
+
+Mit der freien Software *ocelot.social* und dem nötigen Know-How kannst du ein eigenes Netzwerk betreiben (→ [Dokumentation](https://docs.ocelot.social/deployment/), auf Englisch).
+
+Benötigst du dabei Hilfe, dann [unterstützen](/de/contact/) dich unsere Entwickler gerne.
+
+### Entwickle und unterstütze die Software
+
+#### Mach die *ocelot.social*-Software bekannt
+
+*›Spread the word!‹*
+
+Ob Verein, Bewegung oder Institution – *ocelot.social* kann als soziales Netzwerk für viele Arten von Gruppen und Organisationen eingesetzt werden.
+
+- Veröffentliche den Link zu *ocelot.social* auf deiner Website, deinem Blog oder in sozialen Netzwerken.
+- Gib der Software *ocelot.social* auf der Vergleichsplattform [alternative.to](https://alternativeto.net/software/ocelot-social/about/) ein Like.
+- Gib der Software auf der Entwicklerplattform [GitHub](https://github.com/Ocelot-Social-Community/Ocelot-Social) einen Stern.
+
+#### Verbessere das Netzwerk
+
+- Hast du einen Fehler im Netzwerk gefunden, dann melde ihn unseren Entwicklern (→ [Anleitung im Wiki](https://github.com/Ocelot-Social-Community/Ocelot-Social/wiki/de:FAQ#wie-kann-ich-einen-fehler-des-netzwerks-melden)).
```

Comment:
- this sentence should clearly and simply refer to the [issue section of the Ocelot social repository](https://github.com/Ocelot-Social-Community/Ocelot-Social/issues).
- the provided e-mail address in the [wiki link](https://github.com/Ocelot-Social-Community/Ocelot-Social/wiki/de:FAQ#wie-kann-ich-einen-fehler-des-netzwerks-melden) does not exist or work (might be an issue for the [Ocelot-Social repo](https://github.com/Ocelot-Social-Community/Ocelot-Social)).

Reply:
- lets talk about the sentence
- not working e-mail address: see above